tools/project_manager/page_detail.php

```diff
@@ -43,18 +47,61 @@
     $round_for_page_selection = null;
 }
 
-$project = new Project($projectid);
 $state = $project->state;
 $title = $project->nameofwork;
 $page_details_str = _('Page Detail');
 
 output_header("$page_details_str: $title", NO_STATSBAR);
 
+// NB We use a flexbox to position the linkbox div rather than just using
+// `float: right` like the other linkboxes in the codebase do because on
+// this page the inflexible and very wide page details table causes the
+// linkbox to display in ugly locations.
+echo "<div style='display: flex; justify-content: space-between'>";
+echo "<div>";
 echo "<h1>" . html_safe($title) . "</h1>\n";
 echo "<h2>$page_details_str</h2>\n";
+echo "</div>";
+if ($may_select_user) {
+    echo "<div>";
```

Comment:
I'll see if I can remove some of the layers, but I needed the top level div-inside-div so that the linkbox didn't get stretched to the same vertical size as the h1/h2 div.

Reply:
OK. One layer of <div> removed.